goldenhour/timelapse/pi.py

```diff
@@ -1,31 +1,44 @@
 import argparse
+import logging
 import os
 import subprocess
 
 from .ffmpeg import compile_video
 
 
+logger = logging.getLogger()
+
+
 def capture(output_dir, duration, interval):
-    print('capturing one photo every {interval} seconds for {duration} seconds'.format(
+    logger.info('capturing one photo every {interval} seconds for {duration} seconds'.format(
         duration=duration,
         interval=interval,
     ))
     output_pattern = '{}/image%05d.png'.format(output_dir)
-    # TODO check exit status
-    subprocess.call([
-        'raspistill',
-        '-t', str(duration * 1000),
-        '-tl', str(interval * 1000),
-        '-n', # don't try to show a preview window
-        '-w', '1280',
-        '-h', '720',
-        #'--rotation', '180', # set this to correct for camera orientation
-        '--quality', '100',
-        # '--verbose',
-        # '--saturation', '50',
-        '-o', output_pattern,
-        '-e', 'png',
-    ])
+    file_list = os.listdir(output_dir)
```

Comment:
It's a little confusing to have (and reuse) the variable `file_list` when we only want its length. Maybe inline it?

Reply:
Done